tests/e2e/support/objects/app-files/resource/actions.ts

```diff
@@ -1645,3 +1665,13 @@ export const openShotcut = async ({
     ])
   }
 }
+
+export interface expectFileToBeLockedArgs {
+  page: Page
+  resource: string
+}
+
+export const isLocked = async (args: expectFileToBeLockedArgs): Promise<Locator> => {
```

Comment:
would be more suitable here
```suggestion
export const getLockLocator = async (args: expectFileToBeLockedArgs): Promise<Locator> => {
```